modules/charon-core/src/test/java/org/wso2/charon/core/decoder/JSONDecoderTest.java

```diff
@@ -177,28 +110,28 @@ public void testDecodeUser() {
             Assert.assertEquals("babs@jensen.org", decodedUser.getEmailByType("home"));
             Assert.assertTrue(decodedUser.getActive());
 
-
-//            try {
-//                Assert.assertEquals("bjensen@example.com", decodedUser.getPrimaryEmail());
-//                decodedUser.getPrimaryEmail();
-//            } catch (NotFoundException e) {
-//                e.printStackTrace();
-//
-//            }
-
-            Assert.assertEquals("555-555-5555", decodedUser.getPhoneNumbers("work").get(0));
-            Assert.assertEquals("555-555-4444", decodedUser.getPhoneNumbers("mobile").get(0));
-            Assert.assertEquals("someaimhandle", decodedUser.getIMs("aim").get(0));
-            Assert.assertNull("Since the value of user type was empty, this should not be returned from storage",decodedUser.getUserType());
-            Assert.assertEquals("Tour Guide", decodedUser.getTitle());
-            Assert.assertEquals("en_US", decodedUser.getPreferredLanguage());
-            Assert.assertEquals("en_US", decodedUser.getLocale());
-            Assert.assertEquals("America/Los_Angeles", decodedUser.getTimeZone());
-          //  Assert.assertEquals("true", decodedUser.getActive());
-            Assert.assertEquals("t1meMa$heen", decodedUser.getPassword());
-            Assert.assertEquals("00300000005N2Y6AA", decodedUser.getGroups().get(0));
-            Assert.assertEquals("00300000005N34H78", decodedUser.getGroups().get(1));
-            Assert.assertEquals("00300000005N98YT1", decodedUser.getGroups().get(2));
+            //            try {
+            //                Assert.assertEquals("bjensen@example.com", decodedUser.getPrimaryEmail());
+            //                decodedUser.getPrimaryEmail();
+            //            } catch (NotFoundException e) {
+            //                e.printStackTrace();
+            //
+            //            }
+
```

Comment:
Will the above commented try catch block important in future?
Otherwise can you remove it from the code?